pkg/sql/pgwire/conn.go

```diff
@@ -321,6 +321,44 @@ Loop:
 		typ, n, err = c.readBuf.ReadTypedMsg(&c.rd)
 		c.metrics.BytesInCount.Inc(int64(n))
 		if err != nil {
+			// Only perform this logic on ClientMsgSimpleQuery for v20.2.
+			// We cannot safely do the rest of this (i.e. for portals) without doing
+			// the todo message by jordan regarding error in extended protocol state
+			// below.
+			if pgwirebase.IsMessageTooBigError(err) && typ == pgwirebase.ClientMsgSimpleQuery {
+				log.VEventf(ctx, 1, "pgwire: found big error message; attempting to slurp bytes and return error: %s", err)
```

Comment:
The use of `VEventf` is not really warranted here and below -- we don't typically collect nor look at pgwire traces.

I think you can go for `log.Infof` under `log.V(1)`

(if you find yourself wishing for `if log.V(1) { log.Infof() } ` to be a thing without the tracing cruft, feel free to add the wrapper in the `log` package)

Reply:
Done